### DISCUSSION TITLE
Availability on conda?

### DISCUSSION THREAD

**Discussion by TangoTom** [luolingchun/flask-openapi3#46] [Q&A] [answered]
Hi,

just started to use this package and like it.

Unfortunately my company doesn't like pip and so I searched for flask-openapi3 in conda channels without result.

Will this packge be published to a conda channel?

Thanks, Thomas

**Comment by luolingchun** [comment]
Hey @TangoTom I notice your issue [#21091](https://github.com/conda-forge/staged-recipes/issues/21091) and i create a PR [#21248](https://github.com/conda-forge/staged-recipes/pull/21248).

We can focus on it.

**Comment by luolingchun** [comment, marked as answer]
Now, it's available at https://anaconda.org/conda-forge/flask-openapi3.
